FushaarProvider/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+version = 1
+
+cloudstream {
+    description = ""
+    authors = listOf( "ImZaw" )

```

Comment:
Authors are the people who created/edited the provider, so i think it should be Spoonge